app/services/payment/spam_protection_service.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Payment
+  class SpamProtectionService
+    def initialize(account, payment_status, user_session = nil)
+      @account            = account
+      @payment_successful = payment_status
+      @gateway_setting    = account.gateway_setting
```

Comment:
We are not using this at all, no?

![Kill it with fire](https://gph.is/1jpVPVk)